src/Workspaces/Core/Portable/FindSymbols/DeclaredSymbolInfo.cs

```diff
@@ -105,42 +106,42 @@ internal enum DeclaredSymbolInfoKind : byte
             Span = span;
             InheritanceNames = inheritanceNames;
 
-            const uint MaxFlagValue = 0b1111;
+            const uint MaxFlagValue = 0b10000;
```

Comment:
because you're chaning the format here, need to rev teh version number in SyntaxTreeIndex.SerializationFormatChecksum.  That way we don't read out persisted data from a previous version that odesn't match our current format.